src/plugins/index.js

```diff
@@ -18,6 +18,7 @@ import createIframePlugin from './iframePlugin';
 import createFootnotePlugin from './footNotePlugin';
 import createConceptPluin from './conceptPlugin';
 import createRelatedContent from './relatedContentPlugin';
+import codePlugin from './codePlugin';
```

Comment:
For consistency så kan denne gjerne kalles `createCodePlugin`. Om du gidder hadde det også vært fint å fått renamet `createConceptPluin` til `createConceptPlugin` 😂

Reply:
Haha. Det skal eg gjøre!